inst/ms/aggreCAT.qmd

```diff
@@ -536,27 +565,26 @@ an exemplar aggregation method from each mathematical aggregator
 #| prompt: true
 set.seed(1234)
 focal_claims <- data_ratings %>% 
-  filter(paper_id %in% c("24", "138", "186", "108"))
+  dplyr::filter(paper_id %in% c("24", "138", "186", "108"))
```

Comment:
This is only four papers, what happened to the fifth one? Or should the earlier text be updated?

Reply:
- [ ] @eggouldo to update earlier text